generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/string/OfLength.feature

```diff
@@ -444,3 +444,15 @@ Feature: User can specify the length of generated string data using 'ofLength'
       | foo  |
       | null |
       | "1"  |
+
+  Scenario: ofLength with maximum permitted value should be successful
+    Given foo is of length 1000
+    And the generator can generate at most 1 rows
+    And the generation strategy is random
+    And foo is anything but null
+    Then foo contains strings of length between 1000 and 1000 inclusively
```

Comment:
For completeness, other errors include the field name and detail of what the value is, could we do this here?

See one of the other scenarios, e.g. `shorterThan 0`